code/modules/mob/living/carbon/give.dm

```diff
@@ -0,0 +1,58 @@
+/mob/living/verb/give()
```

Comment:
Put this in /haven/code/modules/etc... instead of /code/modules/etc... 

Also, there's a lot of whitespace you could get rid of.
And, as was said, include it in the .dme. Send me a ping on discord if you're not sure how to do that.